### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by mathbunnyru**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+<!-- Fix: https://github.com/jupyter/docker-stacks/issues/0 -->
+
+## Checklist (especially for first-time contributors)
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests
+- [ ] I will try not to use force-push to make the review process easier for reviewers
+
+<!-- markdownlint-disable-file MD041 -->
```
Comment (mathbunnyru) [inline .github/pull_request_template.md:13]: This is a better way to hide these comments, so people do not see these comments when they start reading this file while filling the PR template.

**Comment by mathbunnyru** [comment]
@consideRatio may I ask you to take a look here, please?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Describe your changes
+
+## Issue ticket if applicable
+
+<!-- Fix: https://github.com/jupyter/docker-stacks/issues/0 -->
```
Comment (trallard) [inline .github/pull_request_template.md:5]:
```suggestion
<!-- Example - Fix: https://github.com/jupyter/docker-stacks/issues/0 -->
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Checklist (especially for first-time contributors)
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests
+- [ ] I will try not to use force-push to make the review process easier for reviewers
+
```
Comment (trallard) [inline .github/pull_request_template.md:12]: What about adding:

- [ ] I have updated the documentation for significant changes

**Review by trallard — COMMENTED** [review]
Made some minor comments, otherwise it looks good

**Pull request merged**